Annotation/Service.php

```diff
@@ -53,4 +53,10 @@
 
     /** @var array<string> */
     public $environments = array();
+
+    /** @var boolean */
+    public $autowire;
+
+    /** @var array<string> */
+    public $autowiring_types = array();
```

Comment:
Indeed, I'll switch this one and prepare an additional PR with the rename/deprecation of `decoration_inner_name` (it seems important enough to warrant its own PR.)